cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -191,6 +191,8 @@ func TestEvaluateExecute(t *testing.T) {
 			Arguments: []string{
 				"--language", "golang",
 				"--model", "symflower/symbolic-execution",
+				"--repository", filepath.Join("golang", "plain"),
+				"--repository", filepath.Join("java", "plain"),
```

Comment:
Why is `java/plain` listed if the language is `golang`?

Reply:
errrrrrr... yes I think we can get rid of that one...